python/src/trezorlib/binance.py

```diff
@@ -14,27 +14,39 @@
 # You should have received a copy of the License along with this library.
 # If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
 
+from typing import TYPE_CHECKING
+
 from . import messages
 from .protobuf import dict_to_proto
 from .tools import expect, session
 
+if TYPE_CHECKING:
+    from .client import TrezorClient
+    from .tools import Address
+
 
 @expect(messages.BinanceAddress, field="address")
-def get_address(client, address_n, show_display=False):
+def get_address(
+    client: "TrezorClient", address_n: "Address", show_display: bool = False
+) -> str:
```

Comment:
In this case it does not check the `str` (I tried to change it to something else and the check was still fine). Currently also the return value from `expect.__call__.wrapped_f` is `Any`, so it could be fine with anything. I tried to change that return value to something else (also in `__call__`), but the checks were still successful.

In `core` `setup.cfg` we have an option `disallow_untyped_decorators = True`, so it may help.